src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -392,6 +396,49 @@ export function useServicesValidation(): StepValidation {
   };
 }
 
+function validateUserName(userName: string): string {
+  if (!isUserNameValid(userName)) {
+    return 'Invalid user name';
+  }
+  return '';
```

Comment:
It seems like the purpose of these functions is to generate the error message. If there is no error message, would it make more sense to return `undefined`?

Reply:
no, because if I will return undefined, then I need to change `
export type StepValidation = {
  errors: {
    [key: string]: string| undefined;
  };
  disabledNext: boolean;
};`
and I think it is not make more sense